src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -143,6 +143,30 @@ public void SetAccessControl_FileInfo_FileSecurity_Success()
             fileInfo.SetAccessControl(fileSecurity);
         }
 
+        [Fact]
+        public void SetAccessControl_FileInfo_FileSecurity_Success_NameLongerThanMaxShortPath()
+        {
+            using var directory = new TempAclDirectory();
+
+            const int MaxShortPath = 260;
+            int fileNameLength = MaxShortPath - directory.Path.Length;
+
+            if (fileNameLength < 1)
+            {
+                fileNameLength = 1;
+            }
```

Comment:
nit: this can be simplified

```suggestion
            int fileNameLength = Math.Max(MaxShortPath - directory.Path.Length, 1);
```

Reply:
> nit: this can be simplified

Sure. let me commit.